Mesh_3/include/CGAL/Mesh_3/initialize_triangulation_from_labeled_image.h

```diff
@@ -114,28 +127,50 @@ void initialize_triangulation_from_labeled_image(C3T3& c3t3,
                                              image.vy()),
                                   image.vz());
 
-  typedef std::vector<std::pair<Bare_point, std::size_t> > Seeds;
+  using Point_indices = std::tuple<std::size_t, std::size_t, std::size_t>;
+  using Seed = std::pair<Point_indices, std::size_t>;
```

Comment:
I think that would be more readable to use a plain `struct` instead of a pair with a tuple inside:
```suggestion
  struct Seed {
    std::size_t i, j, k;
    std::size_t radius;
  };
```

That would avoid all the extra syntax with `get<0>`, `get<1>`, `.first`, `.second`.